.github/workflows/regression.yml

```diff
@@ -16,10 +16,10 @@ jobs:
         python-version: ['3.7', '3.8', '3.9', '3.10']
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v2
 
     - name: Setup Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v3.1.2
+      uses: actions/setup-python@v2
```

Comment:
why are we using `v2` here according to https://github.com/actions/setup-python we are at v4